app/views/static_pages/dashboard_steps/moderation_matrix.md

```diff
@@ -11,7 +18,8 @@ This matrix is not all encompassing and zaps are given at the discretion of the
 * Asking about or encouraging the use of Windows
 * "Chat bombing", aka posting many short (ususally 4-5 or fewer words) messages in rapid succession, repeatedly sending parts of thoughts (incomplete, non-standalone sentences or fragments) in new messages. (Use shift + enter and send ONE message where possible or edit your message)
 * Discussing mental health issues in depth or seeking mental health help from other users
-* Inappropriate usernames and profile pictures (E.g. suggestive or inappropriate memes)
+* Inappropriate User Profile - Profile Picture, Username/Nickname, Bio (E.g. suggestive or inappropriate memes)
+* "Crossposting", aka posting the same message to different channels (asking for help in multiple channels at once)
 
 #### 1 Zap (WARNING)
 
```

Comment:
This might be out of scope of this PR but this is the only zap tier that explicitly states that repeat zap tier offences move up. Line 76 (or 84, I am not sure...) offers this as the explanation:

`Repeated offenses increase to the next penalty tier.`

I would propose moving it to the introduction section of the Matrix and removing `A second infraction of any 0 Zap issues above` from here. Alternatively, change the wording to `A second infraction of any [zap tier] issues, including those moved up previously` or something along these lines and putting it in every zap tier.

Reply:
Good point. I am going to have to think on this one before I act on it.